.github/workflows/bench.yml

```diff
@@ -11,30 +11,18 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.7
-        uses: actions/setup-python@v3
+      - name: Set up ssh
+        uses: webfactory/ssh-agent@v0.7.0
         with:
-          python-version: 3.7
-      - name: Install dependencies
-        uses: ./.github/actions/linux_install
-      - name: Install python dependencies
-        uses: ./.github/actions/pip_installation
-      - name: Install python benchmark dependencies
+            ssh-private-key: ${{ secrets.SSH_DEPLOY_KEY }}
+      - name: Clone pyccel-benchmarks repository
+        working-directory: ../
         run: |
-          python -m pip install pythran
-          python -m pip install numba
-          python -m pip install pyperf
-      - name: Benchmark
+          git clone git@github.com:pyccel/pyccel-benchmarks.git
+      - name: Push results to pyccel-benchmarks
+        working-directory: ../pyccel-benchmarks
         run: |
-          python benchmarks/run_benchmark.py --pyperf --verbose
-          echo "# Performance Comparison (as of $(date))" > performance.md
-          cat bench.out >> performance.md
-        shell: bash
-        working-directory: ./.
-      - name: Add & Commit
-        uses: EndBug/add-and-commit@v9.0.0
-        with:
-          message: 'Update performance comparison'
-          add: 'performance.md'
-          default_author: github_actions
+          git config user.email "yaman.guclu@gmail.com"
+          git config user.name "Yaman Güçlü"
+          git commit --allow-empty -m "Benchmark of pyccel@${GITHUB_SHA}"
```

Comment:
I've just checked and GitHub generates the wrong hyperlink from `GITHUB_SHA`. 

For example, in commit [459c58b](https://github.com/pyccel/pyccel-benchmarks/commit/459c58b9fef51989cf2fc4feffde67fc7d05174f), instead of [this correct link](https://github.com/pyccel/pyccel/commit/dfeca799e36bc0f44ce290bc6e8b94fb5361b94c) it creates [this wrong link](https://github.com/pyccel/pyccel-benchmarks/commit/dfeca799e36bc0f44ce290bc6e8b94fb5361b94c).

We probably need to be more specific when we construct this link.

Reply:
Fixed [381e485](https://github.com/pyccel/pyccel/pull/1234/commits/381e485c7bd24a02b3b254a60dfc40eeb0c03283)